syntax = "proto3";

package protobuf;

service StreamService {
  rpc FetchResponse (streamRequest) returns (stream streamResponse) {}
}

message streamRequest {
}

message streamResponse {
  string postid = 1;
  string title = 2;
  int64 score = 3;
  string author = 4;
  string removed = 5;
  int64 total_awards = 6;
  string awarders = 7;
  string created_utc = 8;
  string full_link = 9;
  int64 num_comments = 10;
  string over_18 = 11;
}